src/rttest.cpp

```diff
@@ -12,30 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <rttest.h>
```

Comment:
This as well as `utils.h` should not be importable without a namespace / folder in the first place.

https://github.com/ros2/rttest/blob/2db2779fbdff6934797d645012ff2bb302e99dd5/CMakeLists.txt#L18 should not point to `include/rttest` but to `include`.